timed/redmine/tests/test_redmine_report.py

```diff
@@ -83,3 +83,40 @@ def test_redmine_report_invalid_issue(db, freezer, mocker, capsys, report_factor
 
     _, err = capsys.readouterr()
     assert "issue 1000 assigned" in err
+
+
+def test_redmine_report_calculate_total_hours(
+    db, freezer, mocker, task, report_factory
+):
+    redmine_instance = mocker.MagicMock()
+    issue = mocker.MagicMock()
+    redmine_instance.issue.get.return_value = issue
+    redmine_class = mocker.patch("redminelib.Redmine")
+    redmine_class.return_value = redmine_instance
+
+    freezer.move_to("2017-07-15")
+    reports = report_factory.create_batch(10, task=task)
+
+    freezer.move_to("2017-07-24")
+    reports_last_seven_days = report_factory.create_batch(10, task=task)
+
+    total_hours_last_seven_days = 0
+    for report in reports_last_seven_days:
+        total_hours_last_seven_days = (
```

Comment:
Use `+=` operator to make it more readable

Reply:
Ah yes, forgot about that